sources/api/migration/README.md

```diff
@@ -109,28 +105,28 @@ A reboot will automatically switch back to the other partition set containing th
 Migrations are Rust code adhering to a Migration interface.
 This is defined in the `migration-helpers` library; see [Helpers](#helpers).
 
-The interface will require handling forward and backward migrations through specific methods.
+The interface requires handling forward and backward migrations through specific methods.
 Each method will give data and metadata maps as input, and require data and metadata maps as output.
-(These structures will be nested, rather than using dotted keys, to make it easier to handle sub-trees and reduce dependency on existing data store code.)
+(These structures are nested, rather than using dotted keys, to make it easier to handle sub-trees and reduce dependency on existing data store code.)
 
 Migration code should not assume that any given keys exist, because migrations will be run on live data (where all keys will likely exist) and on pending data (where none, some, or all keys may exist).
 Plus, different variants of Bottlerocket may not have the same keys.
 
-The migration system could deserialize the function outputs into the incoming model types to confirm that the structure is valid; we should prototype this idea because it would add safety.
+The migration system deserializes the function outputs into the incoming model types to confirm that the structure is valid.
```

Comment:
This was more of a TODO comment; my sense is that it's not implemented.

Reply:
Ah. Gotcha. Inline it's not super clear that it's a todo. 

What if we change the **Open questions** heading to **Open questions and future directions** and move this down to that section? That way all the maybe stuff goes into end of the document.

Reply:
Changed in [73d2b39](https://github.com/bottlerocket-os/bottlerocket/pull/2141/commits/73d2b39eebdc48fa6dd7d77bae97207342828693)